pycoal/mineral.py

```diff
@@ -362,3 +365,99 @@ def convert(cls, data_dir="", db_file="", hdr_file=""):
         library = aster_database.create_envi_spectral_library(spectrum_ids, band_info)
 
         library.save(hdr_file)
+
+class SpectralToAsterConversion:
+    
+    def __init__(self):
+        """
+            This class provides a method for converting USGS Spectral Library Version 7
+            <https://speclab.cr.usgs.gov/spectral-lib.html> .txt files into ASTER Spectral
+            Library Version 2.0 <https://asterweb.jpl.nasa.gov/> .txt files
+            
+            Args:
+                none
+            """
+        pass
+    
+    @classmethod
+    def convert(cls, library_filename=""):
```

Comment:
This code is duplicate as that from ```examples/convert_to_aster.py```. If also suffers from the same issues I've pointed out in my review of that file... correct and resubmit a PR.